.NET/Microsoft.Recognizers.Definitions.Common/English/TimeZoneDefinitions.cs

```diff
@@ -527,6 +534,10 @@ public static class TimeZoneDefinitions
             { @"atlantic standard", -240 },
             { @"aus central standard", 570 },
             { @"aus eastern standard", 600 },
+            { @"australian eastern", 600 },
+            { @"australian eastern standard", 600 },
+            { @"australian central daylight", 630 },
+            { @"australian eastern daylight Time", 660 },

```

Comment:
Why remove "@"central daylight", -300 }," and "@"eastern daylight", -240" from a previous commit?
These names are still timezones we want to catch, but their resolution should be -1000, as they are ambiguous.

Reply:
In case of such conflicts, the YAML file should contain both definitions, but commented out (with '#'). You can see some examples in the acronym resolution list.